web/template/layout/data-loss-warning-dialog.gohtml

```diff
@@ -0,0 +1,18 @@
+{{ define "data-loss-warning-dialog" }}
+    <div id="dialog-overlay" class="govuk-!-display-none" tabindex="-1"></div>
```

Comment:
you could incorporate the overlay in the modal so only one thing has to be toggled

Reply:
Nice, hadn't thought of that. Will move it up.